frontend/www/js/omegaup/components/course/TemporaryProblemList.vue

```diff
@@ -0,0 +1,219 @@
+    <div class="card-header">
+      <h5>
+        {{ T.courseAddProblemsAdd }}
+      </h5>
+      <span>{{ T.courseAddProblemsAddAssignmentDesc }}</span>
+    </div>
+    <div class="card-body">
+      <div class="empty-table-message" v-if="problems.length == 0">
+        {{ T.courseAssignmentProblemsEmpty }}
+      </div>
+      <div v-else="">
+        <table class="table table-striped">
+          <thead>
+            <tr>
+              <th>{{ T.contestAddproblemProblemName }}</th>
+              <th>{{ T.contestAddproblemProblemPoints }}</th>
+              <th>{{ T.contestAddproblemProblemRemove }}</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-for="problem in problems">
+              <td>{{ problem.alias }}</td>
+              <td>{{ problem.points }}</td>
+              <td class="button-column">
+                <button
+                  class="btn btn-link"
+                  v-bind:title="T.courseAssignmentProblemRemove"
+                  v-on:click.prevent="onRemoveProblem(assignment, problem)"
+                >
+                  <font-awesome-icon icon="trash" />
+                </button>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+    <div class="card-footer">
+      <form>
+        <div class="row">
+          <div class="col-md-12">
+            <div class="row">
+              <div class="form-group col-md-8">
+                <label
+                  >{{ T.wordsProblem }}
+                  <omegaup-autocomplete
+                    class="form-control"
+                    v-bind:init="(el) => typeahead.problemTypeahead(el)"
+                    v-model="problemAlias"
+                  ></omegaup-autocomplete
+                ></label>
+                <p class="help-block">
+                  {{ T.courseAddProblemsAssignmentsDesc }}
+                </p>
+              </div>
+              <div class="form-group col-md-4">
+                <label
+                  >{{ T.wordsPoints }}
+                  <input type="number" class="form-control" v-model="points" />
+                </label>
+              </div>
+            </div>
+            <div class="form-group text-right">
+              <button
+                data-add-problem
+                class="btn btn-primary mr-2"
+                type="submit"
+                v-bind:disabled="problemAlias.length == 0"
+                v-on:click.prevent="
+                  onAddProblem(assignment, {
+                    alias: problemAlias,
+                    points: points,
+                  })
+                "
+              >
+                {{ T.courseEditAddProblems }}
+              </button>
+            </div>
+          </div>
+        </div>
+      </form>
+    </div>
+    <!-- card-body -->
+  </div>
+  <!-- card -->
+</template>
+
+<style lang="scss" scoped>
+.omegaup-course-problemlist .form-group > label {
```

Comment:
```suggestion
.form-group > label {
```

el `scoped` ya se encarga de agregar un atributo único en la raíz, así que no es necesaria esa clase. de paso puedes quitársela a L2?

Reply:
Listo, sólo tendré que ajustar las pruebas de selenium en el otro cambio, ya que estas hacen referencia a la clase que se quitará.

Reply:
va, igual esas pruebas se debieron haber ajustado para evitar referenciar clases en Selenium.

Reply:
Si, poco a poco hemos estado quitando muchas clases referenciadas, lo cual hace mas legibles las pruebas.